OpenRA.Mods.Common/Widgets/UnitCommandWidget.cs

```diff
@@ -158,7 +168,8 @@ bool PerformStanceCycle()
 				return new Order("SetUnitStance", a, false) { ExtraData = (uint)nextStance };
 			});
 
-			Game.AddChatLine(Color.White, "Battlefield Control", "Unit stance set to: {0}".F(nextStance));
+			if (cycleStances)
```

Comment:
After changing stance, I don't expect experienced player will check it in chat so for me it is clutter. There are more important messages to catch, however it seems it is preferred so I add the message  back.